src/annotator/test/guest-test.js

```diff
@@ -1451,7 +1451,7 @@ describe('Guest', () => {
     assert.isFalse(sidebarRPC().call.calledWith('documentInfoChanged'));
 
     emitSidebarEvent('featureFlagsUpdated', {
-      book_as_single_document: true,
+      some_new_feature: true,
```

Comment:
Can you help me understand the purpose of the dummy feature here?

Reply:
It is just meant to illustrate what a realistic event would look like.